code/modules/mob/mob_defines.dm

```diff
@@ -203,5 +203,12 @@
 
 	///THe z level this mob is currently registered in
 	var/registered_z = null
-	
+
 	var/memory_throttle_time = 0
+
+//austation begin -- fuck byond i cant modularize this for some reason
+
+	/// Used for tracking last uses of emotes for cooldown purposes
+	var/list/emotes_used
+
+//austation end
```

Comment:
the reason you can't modularize this is because the proc that uses the variable is included before byond knows that this variable exists. Once you move everything to their own files you can modularize this too